tests/pl/light_HassLightSet.yaml

```diff
@@ -2,31 +2,56 @@ language: pl
 tests:
   # brightness
   - sentences:
-      - "Ustaw jasność sypialni na 50%"
       - "Ustaw jasność w sypialni na 50 procent"
     intent:
       name: "HassLightSet"
       slots:
         brightness: 50
         area: "bedroom"
         name: all
+    response: Ustawiono jasność w sypialni na 50
```

Comment:
Tu nie powinno być napisane "procent"?

Reply:
In the response you have this sentences:
```
language: pl
responses:
  intents:
    HassLightSet:
      brightness: Ustawiono jasność {{ slots.name }} na {{ slots.brightness }}
      brightness_area: Ustawiono jasność w {{ slots.area }} na {{ slots.brightness }}
      color: Ustawiono kolor {{ slots.name }} na {{ slots.color }}
      color_area: Ustawiono kolor w {{ slots.area }} na {{ slots.color }}
```
again we have a problem with sentence without units. I don't know how to discover unit and in documentation https://developers.home-assistant.io/docs/intent_builtin/ I can't add custom slot name to try define unit.